### PR TITLE
Play upgrade

### PR DESCRIPTION
**Pull request by Jodlowy**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,18 @@
 sudo: false
 language: scala
-scala:
-- 2.11.7
 jdk:
-- oraclejdk8
+  - openjdk8
```
Comment (daniel-manning) [inline .travis.yml:4]: Just delete the travis file, we don't use it

File: build.sbt [hunk 2]
```diff
@@ -1,22 +1,33 @@
...
     name := appName,
     scalaVersion := "2.11.12",
-    crossScalaVersions := Seq("2.11.12"),
+    crossScalaVersions := Seq("2.11.12", "2.12.10"),
     libraryDependencies ++= AppDependencies.all,
     resolvers += Resolver.typesafeRepo("releases"),
     resolvers += Resolver.bintrayRepo("hmrc", "releases"),
-    makePublicallyAvailableOnBintray := true
+    makePublicallyAvailableOnBintray := true,
+    PlayKeys.playPlugin := true
```
Comment (daniel-manning) [inline build.sbt]: Can you explain what this gives you?

**Review by daniel-manning — COMMENTED** [review]
otherwise looks fine, tested within another project

File: .travis.yml (hunk 1 above)
Comment (shaileshp0110) [reply .travis.yml:4]: Travis file is required for the open-source library it's for the Travis builds to publish the library 
https://travis-ci.org/hmrc/play-language/builds/596028100?utm_source=github_status&utm_medium=notification @daniel-manning

File: build.sbt (hunk 2 above)
Comment (Jodlowy) [reply build.sbt]: It's actually no longer required so I'll remove. Thanks for pointing out @daniel-manning

File: .travis.yml (hunk 1 above)
Comment (daniel-manning) [reply .travis.yml:4]: We don't use travis any more, it's not supported. Slack bot will tell you all about it if you ask.

Comment (shaileshp0110) [reply .travis.yml:4]: ok if this is the case then we can delete it

Comment (shaileshp0110) [reply .travis.yml:4]: @daniel-manning 
https://confluence.tools.tax.service.gov.uk/display/TEC/2019/07/02/Non-library+release+packages+will+be+deleted+from+Bintray
Travis build is still required for the library it's only not applicable to service in open, and play language is a library.

Comment (daniel-manning) [reply .travis.yml:4]: It's not worth having an argument, I'll have someone else delete it later.

**Pull request merged**